### PR TITLE
ci-tests: dbench test suite for nfs-ganesha client

### PR DESCRIPTION
**Pull request by grajoria**
Added client script to run dbench test suite on v3, v4 and
v4.1 mount for nfs-ganesha client.

Signed-off-by: Girjesh Rajoria <grajoria@redhat.com>

### CODE REVIEW

File: dbench/client.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+if [ $status -eq 0 ]
+then
+      tail -21 ../dbenchTestLog.txt
+      echo "dbench Test: SUCCESS"
+else
+      tail -5 ../dbenchTestLog.txt
+      echo "dbench Test: FAILURE"
+      exit $status
+fi
+
```
Comment (dang) [inline dbench/client.sh]: This should probably unmount the directories as well.

File: dbench/client.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+else
+      tail -5 ../dbenchTestLog.txt
+      echo "dbench Test: FAILURE"
+      exit $status
+fi
+      
+      
+# v4 mount
+mkdir -p /mnt/nfsv4
+mount -t nfs -o vers=4 ${SERVER}:${EXPORT} /mnt/nfsv4
```
Comment (dang) [inline dbench/client.sh]: This should be 4.0, to avoid ambiguity, and because different kernel versions try minor versions in different orders.

**Review by dang — COMMENTED** [review]
Looks good.  A few minor nits.

**Comment by grajoria** [comment]
Hi @dang 
Made the changes. Please review it again.

**Review by dang — APPROVED** [review]

File: dbench/client.sh [hunk 3]
```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+#
+# Environment variables used:
+#  - SERVER: hostname or IP-address of the NFS-server
+#  - EXPORT: NFS-export to test (should start with "/")
+
+echo "Client Script"
```
Comment (soumyakoduri) [inline dbench/client.sh:7]: nit: Was it intentional?

**Review by soumyakoduri — APPROVED** [review]

File: dbench/client.sh (hunk 3 above)
Comment (grajoria) [reply dbench/client.sh:7]: Added this for testing purpose as this avoided scrolling log file to find start of client script. Should I remove this and send PR again?

Comment (soumyakoduri) [reply dbench/client.sh:7]: Thats okay.. was just curious :)

**Pull request merged**